recipes-extended/seabios/patches/only-boot-selected-devices.patch

```diff
@@ -71,11 +75,11 @@ PATCHES
 -                  , desc ? MAKE_FLATPTR(seg, desc) : "Unknown");
 -    DefaultBEVPrio = DEFAULT_PRIO;
 +    if (SelectedBEV) {
-+        bootentry_add(IPL_TYPE_BEV, defPrio(prio, DefaultBEVPrio)
-+                      , SEGOFF(seg, bev).segoff
-+                      , desc ? MAKE_FLATPTR(seg, desc) : "Unknown");
-+        DefaultBEVPrio = DEFAULT_PRIO;
-+    }
++		bootentry_add(IPL_TYPE_BEV, defPrio(prio, DefaultBEVPrio)
++					  , SEGOFF(seg, bev).segoff
```

Comment:
Your refreshed patch introduces hardtabs.  Please revert them to spaces to match the rest of the file.  The changes add noise and makes reviewing a little harder.  Thanks.

Reply:
ack - that was not intended. I've just refreshed this branch to with the hardtabs removed.